maptivist/src/main/webapp/script.js

```diff
@@ -95,6 +100,17 @@ var GoogleAuth;
     }
   }
 
+  function handleAuthClick2() {
```

Comment:
Can you add documentation for the the difference between handleAuthClick and handleAuthClick2?